### DISCUSSION TITLE
Where to find json_build_object function

### DISCUSSION THREAD

**Discussion by asher-feil** [sqlalchemy/sqlalchemy#11181] [Usage Questions]
I'm trying to use the `json_build_object` function but can't seem to find it. I see it referenced in the changelog for 2.0. I was also seeing references to functions like `jsonb_each` and `json_array_elements` in comments for other functions but can't find any of the JSON functions. Were they (re)moved?

I am using version 2.0.28

**Comment by CaselIT** [comment]
Hi,

TLDR `func` can dynamically generate any functions you need.
Please refer to #11155
